package.json

```diff
@@ -10,6 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "mongoose": "^8.1.1"
```

Comment:
It's a good practice to lock the version of your dependencies to avoid unexpected breaking changes in future. You can remove the caret (^) before the version number to lock the version. [medium]